modules/criteoBidAdapter.js

```diff
@@ -17,8 +17,11 @@ const storage = getStorageManager(GVLID);
 const LOG_PREFIX = 'Criteo: ';
 
 // Unminified source code can be found in: https://github.com/Prebid-org/prebid-js-external-js-criteo/blob/master/dist/prod.js
```

Comment:
Is there a plan to update this to include the un-minified publishertag code? This link is broken and the existing prebid-js-external repo is empty. If there isn't we could remove this line.

Reply:
Hi @ncolletti, the link refers to a private repository shared to prebid so needs the access, it's kept here as a hint